conversion/indico_conversion/conversion.py

```diff
@@ -91,6 +92,35 @@ def submit_attachment_doconverter(task, attachment):
         else:
             ConversionPlugin.logger.info('Submitted %r to Doconverter', attachment)
 
+@celery.task(bind=True, max_retries=None)
+def request_pdf_from_googledrive(task, attachment):
+    """Uses the Google Drive API to convert a Google Drive file to a PDF."""  
+    from indico_conversion.plugin import ConversionPlugin
+    ConversionPlugin.logger.info('request_pdf_from_googledrive %r', attachment)
+    # URLS have form: https://docs.google.com/presentation/d/<FILEID>
+    # So look for docs.google.com, and then extract the fileID
+    url_list = attachment.link_url.split('/')
+    try:
+        file_id = url_list[url_list.index('docs.google.com') + 3]
+    except ValueError:
+        return
+
+    # use requests to get the file from this URL: 
+    mime_type = 'application/pdf'
+    api_key = ConversionPlugin.settings.get('googledrive_api_key')
+    request_text = f'https://www.googleapis.com/drive/v3/files/{file_id}/export?mimeType={mime_type}&key={api_key}'
+    ConversionPlugin.logger.info('request_text %r', request_text)
+    try:
+        response = requests.get(request_text)
+        response.raise_for_status()
+        if 'ok' not in response.text:
+            raise requests.RequestException(f'Unexpected response from server: {response.text}', response=response)
```

Comment:
I don't think this makes sense here - Google Drive response with a PDF, not with some weird string ;)

In case of the doconverter system it's async so there we really want to check for this string, but in case of google youg et the PDF straight away...